src/connector/mysql.rs

```diff
@@ -121,6 +131,8 @@ impl MysqlUrl {
         let mut socket_timeout = None;
         let mut connect_timeout = Some(Duration::from_secs(5));
         let mut pool_timeout = Some(Duration::from_secs(10));
+        let mut max_connection_lifetime = None;
+        let mut max_idle_connection_lifetime = None;
```

Comment:
```suggestion
        let mut max_idle_connection_lifetime = Some(Duration::from_secs(300));
```

And remove this in engines https://github.com/prisma/prisma-engines/blob/master/query-engine/connectors/sql-query-connector/src/database/mysql.rs